### PR TITLE
Updated Restore Options List Section with alternative WITH syntax

### LINKED ISSUE #1811: backup/restore: Change to the WITH OPTIONS syntax
PR: https://github.com/cockroachdb/cockroach/pull/17528

@sploiselle, looks like in `RESTORE`, the `WITH OPTIONS` syntax has been changed. According to [this comment](https://github.com/cockroachdb/cockroach/pull/17528#issuecomment-321097489) from @dt, though, the old syntax will continue to work for 1.1, so this is low-priority.

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Closes #1811 

@dt Let me know if the changes are sufficient. Or do we want to replicate the changes in the Examples section as well.

### CODE REVIEW

**Comment by Amruta-Ranade** [comment]
As discussed with @dt, replaced `WITH OPTIONS` examples with `WITH` examples.

File: v1.2/restore.md [hunk 1]
```diff
@@ -121,14 +121,14 @@ You can include the following options as key-value pairs in the `kv_option_list`
...
-- **Example**: `WITH OPTIONS ('into_db' = 'newdb')`
+- **Example**: `WITH into_db = 'newdb'`
 
 #### `skip_missing_foreign_keys`
 
 - **Description**: If you want to restore a table with a foreign key but don't want to restore the table it references, you can [drop the Foreign Key constraint from the table](#skip_missing_foreign_keys) and then have it restored.
 - **Key**: `skip_missing_foreign_keys`
 - **Value**: *No value*
-- **Example**: `WITH OPTIONS ('skip_missing_foreign_keys')`
+- **Example**: `WITH 'skip_missing_foreign_keys'`
```
Comment (dt) [inline v1.2/restore.md]: don't think this needs the quotes

File: v1.2/restore.md [hunk 2]
```diff
@@ -173,7 +173,7 @@ By default, tables with [Foreign Key](foreign-key.html) constraints must be rest
 
 ~~~ sql
 > RESTORE bank.accounts FROM 'azure://acme-co-backup/table-customer-2017-03-27-full?AZURE_ACCOUNT_KEY=hash&AZURE_ACCOUNT_NAME=acme-co'
-WITH OPTIONS ('skip_missing_foreign_keys');
+WITH 'skip_missing_foreign_keys';
```
Comment (dt) [inline v1.2/restore.md]: ditto

File: v1.1/restore.md [hunk 3]
```diff
@@ -121,14 +121,14 @@ You can include the following options as key-value pairs in the `kv_option_list`
...
-- **Example**: `WITH OPTIONS ('into_db' = 'newdb')`
+- **Example**: `WITH into_db = 'newdb'`
 
 #### `skip_missing_foreign_keys`
 
 - **Description**: If you want to restore a table with a foreign key but don't want to restore the table it references, you can [drop the Foreign Key constraint from the table](#skip_missing_foreign_keys) and then have it restored.
 - **Key**: `skip_missing_foreign_keys`
 - **Value**: *No value*
-- **Example**: `WITH OPTIONS ('skip_missing_foreign_keys')`
+- **Example**: `WITH 'skip_missing_foreign_keys'`
```
Comment (dt) [inline v1.1/restore.md]: don't think this need the quotes

**Pull request merged**